packages/admin/admin/src/mui/menu/Item.styles.ts

```diff
@@ -80,23 +82,25 @@ export const Root = createComponentSlot(ListItemButton)<MenuItemClassKey, OwnerS
             color: ${theme.palette.grey[900]};
             height: 60px;
             padding: 16px 22px;
-            background-color: ${!ownerState.open && ownerState.collapsibleOpen ? `${theme.palette.primary.main} !important` : "white !important"};
+            background-color: ${!ownerState.open && ownerState.collapsibleOpen
+                ? `${theme.palette.primary.main} !important`
```

Comment:
Is the important still needed?

Reply:
I don't really want to touch it tbh because if I don't consider every case and forget one we have once again some styling issues so I would just leave it like it is. Rn everything looks fine